schemas/com.sendgrid/bounce/jsonschema/2-0-0

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a SendGrid bounce event",
+  "self": {
+    "vendor": "com.sendgrid",
+    "name": "bounce",
+    "version": "2-0-0",
+    "format": "jsonschema"
+  },
+  "type": "object",
+  "properties": {
+    "ip": {
+      "description": "the IP address used to send the email. For open and click events, it is the IP address of the recipient who engaged with the email.",
+      "type": "string",
+      "format": "ipv4"
```

Comment:
We sure it cannot be ipv6?

Reply:
Yes.

> An Internet Protocol (IP) Address is a unique numerical address that defines an internet location. These addresses have 4 sets of numbers separated by periods. For example, 127.0.0.1 is an IP address.

*Source*: https://sendgrid.com/docs/glossary/ip-address/